sponsorship_compassion/data/gmc_action.xml

```diff
@@ -36,6 +36,20 @@
             <field name="batch_send">1</field>
             <field name="priority">50</field>
         </record>
+        <!-- Sponsorship Message Creation -->
+        <record id="upsert_correspondent_commitment" model="gmc.action">
+            <field name="direction">out</field>
+            <field name="name">UpsertCorrespondentCommitment</field>
+            <field name="mapping_id" ref="mapping_upsert_correspondence_commitment"/>
+            <field name="model">sponsorship.compassion</field>
```

Comment:
This model doesn't exist. Didn't you test the message?
```suggestion
            <field name="model">recurring.contract</field>
```

Reply:
i've tested but i guess i changed on the interface and not in the file (apparently)